site/_utils/drafts.js

```diff
@@ -30,4 +30,30 @@ const filterOutDrafts = item => {
   return true; // include everything in non-prod
 };
 
-module.exports = {filterOutDrafts};
+/**
+ * Checks if the provided date is later than current date and time.
+ *
+ * @param {Date=} postDate Date to be compared.
```

Comment:
```suggestion
 * @param {Date} postDate Date to be compared.
```

Reply:
tsc will complain, as postDate can actually be undefined

Reply:
Got it 🙂